src/main/Domain.Model/IAvatarRepository.cs

```diff
@@ -1,9 +1,14 @@
-﻿using System.Threading.Tasks;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace ei8.Cortex.Subscriptions.Domain.Model
 {
     public interface IAvatarRepository
     {
         Task<Avatar> GetOrAddAsync(string url);
+
+        Task<IList<Avatar>> GetAll();
+
+        Task UpdateAsync(Avatar avatar);
```

Comment:
Please rename Avatar object to AvatarUrl to clarify intent/meaning, also to differentiate it from Avatar (which is in another bounded context, but would make it intuitive.)

https://martinfowler.com/bliki/UbiquitousLanguage.html

Reply:
Hi @instilledbee, I was referring to the class ei8.Cortex.Subscriptions.Domain.Model.Avatar

Perhaps we can change it to something like "AvatarUrlContent" to represent the resulting message body that is received after sending an AvatarUrl request?

Happy to discuss further via zoom/chat.

Reply:
@elmerbool As discussed, renamed all the references to the term **Avatar** into **AvatarUrlSnapshot**. Please see the [latest commit](https://github.com/ei8/cortex-subscriptions/pull/3/commits/c8cb099629bd85fde9f1e704e73253a28adc7907) to compare. Thanks.